tests/e2e/lib/flows/jetpack-connect.js

```diff
@@ -77,7 +77,8 @@ export async function connectThroughWPAdminIfNeeded( {
 	// await ( await JetpackUserTypePage.init( page ) ).selectUserType( 'creator' );
 
 	if ( mockPlanData ) {
-		await persistPlanData();
+		const planType = plan === 'free' ? 'jetpack_free' : 'jetpack_business';
```

Comment:
I am slightly confused about this part as the paid blocks come through this code path, even though this branch calls `.selectFreePlan()` ... will try and work out why that is and see if this can be tidied up.